src/main/resources/finalCopy/final_workflow_log_dir.test

```diff
@@ -0,0 +1,11 @@
+# Tests that the runtime option to preserve logs from the workflow is being honored.
+
+name: workflow_log_dir
+testFormat: workflowsuccess
+backends: [Local]
+tags: [localdockertest]
```

Comment:
backends: [Local] will make it so that this test will only run when Centaur is tested against the Local backend. 

One simple way to test whether copying works in GCS could be checking the workflow-level metadata... I think we publish the location of where workflow logs are copied?